prow/config/config.go

```diff
@@ -3407,7 +3407,7 @@ func sortStringSlice(s []string) []string {
 	return s
 }
 
-func deduplicateTideQueries(queries TideQueries) (TideQueries, error) {
+func (c Config) deduplicateTideQueries(queries TideQueries) (TideQueries, error) {
```

Comment:
Can we make this have a pointer receiver for consistency with other methods?

Reply:
I wasn't sure if consistency was better or making it clear that this method wasn't going to modify the `Config`, but I am fine with it being a pointer.